GSA-TTS/FAC
# Tribal Audit Consent Form

# Tribal Audit Consent/Tribal Data Release Form

Issue: https://github.com/GSA-TTS/FAC/issues/2101

## Changes:
1. Tribal audit consent form
    * The URL `/audit/tribal-data-release/REPORTID`
    * The Django view (its own file in viewlib) that gets the object for display and saves the object on post
    * The HTML for the form
    * The form link in the submission checklist, which is static and does not rely on `submission_progress_check` or anything. 

## How to test:
1. Switch to this branch and run normally
3. Start a submission, or check out an old one
4. Go to the submission checklist and navigate to the tribal audit consent form
5. Fill it out and submit
6. Verify that the object is in the DB with proper variable names

## Screenshots:
Form:
![image](https://github.com/GSA-TTS/FAC/assets/91098850/824deb1a-9147-4712-acae-b7c0805e0350)

Checklist:
![image](https://github.com/GSA-TTS/FAC/assets/91098850/0a6de0ed-63dd-4016-94a5-e124233674f9)

## PR checklist: submitters


## PR checklist: reviewers

        
The larger the PR, the stricter we should be about these points.
